src/main/webapp/index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Black Student Insight</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+    <script src="script.js"></script>
+  </head>
+  <body onload="hideElements()">
+    <header>
+      <nav>
```

Comment:
This section should work fine without this tag.

Reply:
It works but the hideElements() function makes certain elements invisible to the client until they are relevant. So basically, without the function, we will see the little boxes that are normally behind the college-info results in the background if you scroll to the bottom of the home page.